Library/Homebrew/diagnostic.rb

```diff
@@ -532,6 +533,19 @@ def check_user_curlrc
         EOS
       end
 
+      def check_for_unsupported_curl_vars
+        # Support for SSL_CERT_DIR seemed to be removed in the 10.10.5 update.
+        return unless MacOS.version >= :yosemite
+        return if ENV["SSL_CERT_DIR"].nil?
+
+        <<-EOS.undent
+          SSL_CERT_DIR support was removed from Apple's curl.
+          If fetching formulae fails you should:
+            unset SSL_CERT_DIR
+          and remove it from #{Utils::Shell.shell_profile} if present.
+        EOS
+      end
+
```

Comment:
This block of code probably originates from the merge conflict you resolved manually. It should no longer be here, but instead in `extend/os/mac/diagnostic.rb`.